Appraisals

```diff
@@ -2,11 +2,16 @@ appraise "activerecord_4_2" do
   gem "activerecord",  "~> 4.2.0"
   gem "activesupport", "~> 4.2.0"
   gem "sqlite3", "~> 1.3.0"
+
+  # otherwise you get "undefined method `new' for BigDecimal:Class" in Ruby 2.7
+  gem "bigdecimal", "1.3.5"
```

Comment:
We could replace `BigDecimal.new` with `BigDecimal()` all over the code instead. The idea to bring in a gem for it doesn't look right.

Reply:
It's only for testing to be fair, but sure I'll make that change.